packages/transformers/jsonld/README.md

```diff
@@ -0,0 +1,35 @@
+# jsonld transformer
```

Comment:
I think this file can be removed now?

Reply:
We could remove it, but is there harm in keeping it around? Could serve as a nice starter manual for developers working on new transformers? I was really confused when I started working on this and that README is where I put everything I learned. If anything, we could clean it up and move it somewhere else where developer documentation goes.

Up to you.

Reply:
Proper (plugin) docs are in the works. And the part that applies to transformers in general is quite short